Sources/SwiftDocC/Semantics/Metadata/PageImage.swift

```diff
@@ -11,19 +11,26 @@
 import Foundation
 import Markdown
 
+/// A directive to adjust the image associated with the page.
```

Comment:
Could this be simplified by using "associate" as the verb instead?

```suggestion
/// A directive to associate an image with a page.
```